<<<<<<< HEAD
[2024-11-28T23:04:18.453+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T23:04:18.478+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [queued]>
[2024-11-28T23:04:18.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [queued]>
[2024-11-28T23:04:18.488+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-11-28T23:04:18.502+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_openmeteo_data> on 2024-11-28 22:00:00+00:00
[2024-11-28T23:04:18.508+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_kafka_dag', 'store_openmeteo_data', 'scheduled__2024-11-28T22:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream.py', '--cfg-path', '/tmp/tmpih82870n']
[2024-11-28T23:04:18.511+0000] {standard_task_runner.py:91} INFO - Job 224: Subtask store_openmeteo_data
[2024-11-28T23:04:18.511+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=2340) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-28T23:04:18.512+0000] {standard_task_runner.py:64} INFO - Started process 2344 to run task
[2024-11-28T23:04:18.553+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [running]> on host 0741a2a1545f
[2024-11-28T23:04:18.622+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='weather_data_kafka_dag' AIRFLOW_CTX_TASK_ID='store_openmeteo_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-28T22:00:00+00:00'
[2024-11-28T23:04:18.623+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T23:04:18.632+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T23:04:18.633+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/kafka_stream.py", line 150, in store_openmeteo_data
    read_from_kafka_and_store(KAFKA_TOPIC_OPENMETEO, "openmeteo_data.parquet")
  File "/opt/airflow/dags/kafka_stream.py", line 141, in read_from_kafka_and_store
    logging.error(f"Error reading from Kafka topic {topic_name}: {e}")
                                                                  ^
NameError: name 'e' is not defined
[2024-11-28T23:04:18.643+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=weather_data_kafka_dag, task_id=store_openmeteo_data, run_id=scheduled__2024-11-28T22:00:00+00:00, execution_date=20241128T220000, start_date=20241128T230418, end_date=20241128T230418
[2024-11-28T23:04:18.654+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 224 for task store_openmeteo_data (name 'e' is not defined; 2344)
[2024-11-28T23:04:18.686+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-11-28T23:04:18.708+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
=======
[2024-11-28T23:02:06.973+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T23:02:07.021+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [queued]>
[2024-11-28T23:02:07.039+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [queued]>
[2024-11-28T23:02:07.040+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-11-28T23:02:07.064+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_openmeteo_data> on 2024-11-28 22:00:00+00:00
[2024-11-28T23:02:07.083+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1239) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-28T23:02:07.085+0000] {standard_task_runner.py:64} INFO - Started process 1250 to run task
[2024-11-28T23:02:07.088+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_kafka_dag', 'store_openmeteo_data', 'scheduled__2024-11-28T22:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream.py', '--cfg-path', '/tmp/tmprwwkuomq']
[2024-11-28T23:02:07.092+0000] {standard_task_runner.py:91} INFO - Job 35: Subtask store_openmeteo_data
[2024-11-28T23:02:07.168+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_kafka_dag.store_openmeteo_data scheduled__2024-11-28T22:00:00+00:00 [running]> on host 8885fe629c2f
[2024-11-28T23:02:07.321+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='weather_data_kafka_dag' AIRFLOW_CTX_TASK_ID='store_openmeteo_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-28T22:00:00+00:00'
[2024-11-28T23:02:07.324+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T23:02:07.373+0000] {conn.py:362} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.23.0.9', 9092)]>: connecting to broker:9092 [('172.23.0.9', 9092) IPv4]
[2024-11-28T23:02:07.375+0000] {conn.py:957} INFO - Probing node bootstrap-0 broker version
[2024-11-28T23:02:07.377+0000] {conn.py:393} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.23.0.9', 9092)]>: Connection complete.
[2024-11-28T23:02:07.489+0000] {conn.py:1019} INFO - Broker version identified as 2.6.0
[2024-11-28T23:02:07.490+0000] {conn.py:1020} INFO - Set configuration api_version=(2, 6, 0) to skip auto check_version requests on startup
[2024-11-28T23:02:07.493+0000] {consumer.py:122} WARNING - group_id is None: disabling auto-commit.
[2024-11-28T23:02:07.494+0000] {subscription_state.py:167} INFO - Updating subscribed topics to: ('openmeteo_raw',)
[2024-11-28T23:02:07.497+0000] {subscription_state.py:253} INFO - Updated partition assignment: [('openmeteo_raw', 0)]
[2024-11-28T23:02:07.499+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=broker:9092 <connecting> [IPv4 ('172.23.0.9', 9092)]>: connecting to broker:9092 [('172.23.0.9', 9092) IPv4]
[2024-11-28T23:02:07.501+0000] {conn.py:393} INFO - <BrokerConnection node_id=1 host=broker:9092 <connecting> [IPv4 ('172.23.0.9', 9092)]>: Connection complete.
[2024-11-28T23:02:07.503+0000] {conn.py:673} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connected> [IPv4 ('172.23.0.9', 9092)]>: Closing connection. 
[2024-11-28T23:02:07.708+0000] {logging_mixin.py:188} INFO - Consumed 100 records from topic openmeteo_raw
[2024-11-28T23:02:07.814+0000] {logging_mixin.py:188} INFO - Data from openmeteo_raw stored in /opt/***/data/parquet_files/openmeteo_data.parquet
[2024-11-28T23:02:07.815+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-28T23:02:07.816+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T23:02:07.846+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_kafka_dag, task_id=store_openmeteo_data, run_id=scheduled__2024-11-28T22:00:00+00:00, execution_date=20241128T220000, start_date=20241128T230207, end_date=20241128T230207
[2024-11-28T23:02:07.952+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-28T23:02:08.031+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-28T23:02:08.044+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
>>>>>>> 604b8b813e1d85ccda3063889c3dd1126b7c5791
